docs/quickstart.md

```diff
@@ -11,3 +11,59 @@ pulp-operator on top of it.
 Is not considered production ready because pulp-operator is not yet, it hides every config option,
 and upgrades are not considered. Only suitable as a quick way to evaluate Pulp for the time
 being.
+
+
+## OpenShift
+
+Currently pulp-operator is not on the OpenShift catalog, so as first step we need to create a catalog entry:
+
+```yaml
+# pulp-catalog-source.yaml
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+ name: my-pulp-catalog
+ namespace: openshift-marketplace
+spec:
+ sourceType: grpc
+ image: quay.io/pulp/pulp-index:0.2.0
+
+```
+* Refer to [Getting started with the OpenShift CLI](https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/getting-started-cli.html)
+```console
+oc apply -f pulp-catalog-source.yaml
+```
+
+Wait few seconds and refresh OCP page, after that you'll be able to see `my-pulp-catalog`
```

Comment:
```suggestion
Wait few seconds and refresh OCP page, after that you will be able to see `my-pulp-catalog`
```
maybe even use "should" instead of "will".